### PR TITLE
Source Code to PEP8 compliance

### PR DESCRIPTION
**Pull request by AlexanderHaug**

### CODE REVIEW

File: src/mirrulations_client/documents_processor.py [hunk 1]
```diff
@@ -12,21 +12,28 @@
...
+            to the server after each call is processed
     """
     global workfiles
     workfiles = []
     for url in urls:
         try:
             result = api_call_manager(add_api_key(url))
             process_results(result)
-        except:
+        except Exception:
```
Comment (edwharbison) [inline src/mirrulations_client/documents_processor.py]: Use a specific exception!

File: src/mirrulations_client/documents_processor.py [hunk 2]
```diff
@@ -12,21 +12,28 @@
...
             logger.error('Error - URL processing failed')
-    result = json.loads(json.dumps({"job_id" : job_id, "type": "docs", "data" : workfiles, "client_id" : client_id, "version" : version}))
+
+    result = json.loads(json.dumps({"job_id": job_id,
+                                    "type": "docs",
+                                    "data": workfiles,
+                                    "client_id": client_id,
+                                    "version": version}))
```
Comment (edwharbison) [inline src/mirrulations_client/documents_processor.py]: Remove double quotes!

File: src/mirrulations_client/documents_processor.py [hunk 3]
```diff
@@ -66,7 +75,7 @@ def make_docs(doc_list):
             work_list = []
             size = 0
         size += calls
-        document = {"id" : doc_id, "count" : calls}
+        document = {"id": doc_id, "count": calls}
```
Comment (edwharbison) [inline src/mirrulations_client/documents_processor.py]: Remove double quotes!

File: src/mirrulations_core/__main__.py [hunk 4]
```diff
@@ -5,7 +5,8 @@
 import requests
 import string
 
-CONFIG_PATH = os.path.join(os.path.abspath(os.path.dirname(__file__)), '../../config.json')
+CONFIG_PATH = os.path.join(
+    os.path.abspath(os.path.dirname(__file__)), '../../config.json')
```
Comment (edwharbison) [inline src/mirrulations_core/__main__.py]: Split the line at the comma instead if possible.

File: src/mirrulations_core/__main__.py [hunk 5]
```diff
@@ -26,10 +27,13 @@ def config_setup(is_server):
         port = input('Port:\n')
 
     key = input('API Key:\n')
-    client_id = ''.join(random.choices(string.ascii_letters + string.digits, k=16))
+    client_id = ''.join(
+        random.choices(string.ascii_letters + string.digits, k=16))
```
Comment (edwharbison) [inline src/mirrulations_core/__main__.py]: Split the line at the plus and comma instead if possible.

File: src/mirrulations_core/__main__.py [hunk 6]
```diff
@@ -26,10 +27,13 @@ def config_setup(is_server):
...
 
     try:
-        r = requests.get('https://api.data.gov/regulations/v3/documents.json?api_key=' + key)
+        r = requests.get(
+            'https://api.data.gov/regulations/v3/documents.json?api_key=' +
+            key)
```
Comment (edwharbison) [inline src/mirrulations_core/__main__.py]: Split the line at the plus instead if possible, or mid-string.

File: src/mirrulations_core/config.py [hunk 7]
```diff
@@ -10,7 +10,8 @@ def read_value(value):
     :return: Value read from the JSON
     """
     try:
-        configurationpath = os.path.join(os.path.abspath(os.path.dirname(__file__)), "../../config.json")
+        configurationpath = os.path.join(
+            os.path.abspath(os.path.dirname(__file__)), "../../config.json")
```
Comment (edwharbison) [inline src/mirrulations_core/config.py]: Remove double quotes!
Split the line at the comma if possible.

File: src/mirrulations_server/docs_work_gen.py [hunk 8]
```diff
@@ -23,17 +26,20 @@ def monolith():
     if regulations_key != "":
```
Comment (edwharbison) [inline src/mirrulations_server/docs_work_gen.py]: Remove double quotes!

File: src/mirrulations_server/docs_work_gen.py [hunk 9]
```diff
@@ -23,17 +26,20 @@ def monolith():
...
         # Gets number of documents available to download
         try:
-            record_count = requests.get("https://api.data.gov/regulations/v3/documents.json?api_key=" + regulations_key
-                                        + "&countsOnly=1").json()["totalNumRecords"]
-        except:
+            record_count = requests.get(
+                "https://api.data.gov/regulations/v3/documents.json?api_key="
+                + regulations_key
+                + "&countsOnly=1").json()["totalNumRecords"]
```
Comment (edwharbison) [inline src/mirrulations_server/docs_work_gen.py]: Remove double quotes!
See if you can split it better.  This just looks really bad.

File: src/mirrulations_server/docs_work_gen.py [hunk 10]
```diff
@@ -23,17 +26,20 @@ def monolith():
...
+        except Exception:
```
Comment (edwharbison) [inline src/mirrulations_server/docs_work_gen.py]: Use specific exception!

File: src/mirrulations_server/docs_work_gen.py [hunk 11]
```diff
@@ -46,8 +52,11 @@ def monolith():
                 if current_page == max_page_hit:
                     break
 
-            # Makes a JSON from the list of URLs and send it to the queue as a job
-            docs_work = [''.join(random.choices(string.ascii_letters + string.digits, k=16)), "docs", url_list]
+            # Makes a JSON from the list of URLs and send
+            # it to the queue as a job
+            docs_work = [''.join(
+                random.choices(string.ascii_letters + string.digits, k=16)),
+                "docs", url_list]
```
Comment (edwharbison) [inline src/mirrulations_server/docs_work_gen.py]: Split inside `random.choices` at the comma and split `"docs"`, `url_list` at the comma.
Remove double quotes!

File: src/mirrulations_server/docs_work_gen.py [hunk 12]
```diff
@@ -46,8 +52,11 @@ def monolith():
...
             r.add_to_queue(endpoints.generate_json(docs_work))
     else:
         print("No API Key!")
```
Comment (edwharbison) [inline src/mirrulations_server/docs_work_gen.py]: Remove double quotes!

**Comment by bjcoleman** [comment]
I certainly agree with the suggestions about having more specific exceptions.  We should address this, but probably in another PR.

Single quotes are more Pythonic, so I think it makes sense to make those quick changes.

I'm on the fence about the line breaks - no strong opinion either way.

**Comment by edwharbison** [comment]
@bjcoleman 

Understood about the Exceptions.  Will resolve.

PEP8 specifies before operations, not after.  Other than that, I think my Scheme style guide may have been influencing me a bit too much, so I'll resolve those.

File: src/mirrulations_core/config.py (hunk 7 above)
Comment (edwharbison) [reply src/mirrulations_core/config.py]: Don't worry about the line split, still get the double quotes though.

File: src/mirrulations_server/docs_work_gen.py (hunk 9 above)
Comment (edwharbison) [reply src/mirrulations_server/docs_work_gen.py]: Ignore the split thing, but still fix the double quotes.

File: src/mirrulations_server/docs_work_gen.py (hunk 11 above)
Comment (edwharbison) [reply src/mirrulations_server/docs_work_gen.py]: Ignore the split, but remove double quotes.

File: tests/mirrulations_server_tests/test_redis_manager.py [hunk 13]
```diff
@@ -103,7 +103,7 @@ def test_get_specific_item_from_queue_does_not_match():
     r = make_database()
     r.delete_all()
     r.add_to_queue(json.dumps({"A":"B", "job_id":"c"}))
-    assert r.get_specific_job_from_queue("d") == '{"job_id":"null", "type":"none"}'
+    assert r.get_specific_job_from_queue("d") == "{'job_id':'null', 'type':'none'}"
```
Comment (edwharbison) [inline tests/mirrulations_server_tests/test_redis_manager.py]: These should be reverted.  The strings need the doubles inside for the `json` to work right.

File: src/mirrulations_server/redis_manager.py [hunk 14]
```diff
@@ -99,37 +101,40 @@ def get_specific_job_from_queue(self, job_id):
...
             for element in range(self.r.llen('queue')):
-                current = (self.r.lindex('queue', element)).decode("utf-8")
+                current = (self.r.lindex('queue', element)).decode('utf-8')
                 info = json.loads(current)
 
                 if job_id == info['job_id']:
                     return current
 
-            return '{"job_id":"null", "type":"none"}'
+            return "{'job_id':'null', 'type':'none'}"
```
Comment (edwharbison) [inline src/mirrulations_server/redis_manager.py]: These (and similar ones) should be reverted.  The strings need the doubles inside for the `json` to work right.

**Pull request merged**